database/insertData/_common/03_template_C_userRegistration.js

```diff
@@ -131,7 +152,254 @@ exports.queries = [
                         validationMessageInternal: "Password must be at least 8 characters"
                       }
                     }
-                    # TO-DO: Add Date of birth question once we have DatePicker element type
+                    {
+                      code: "PB1"
+                      index: 70
+                      title: "Page Break"
+                      elementTypePluginCode: "pageBreak"
+                      category: INFORMATION
+                    }
+                    {
+                      code: "S1PersonalInfo"
+                      index: 80
+                      title: "Section 1 - Personal information"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: {
+                        title: "## Personal information"
+                        style: "basic"
+                      }
+                    }
+                    {
+                      code: "Q6DOB"
+                      index: 90
+                      title: "DOB"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: { label: "Date of Birth", maxWidth: 150 }
+                      validation: {
+                        operator: "REGEX"
+                        children: [
+                          {
+                            operator: "objectProperties"
+                            children: ["responses.thisResponse"]
+                          }
+                          "^([0]?[1-9]|[1|2][0-9]|[3][0|1])[./-]([0]?[1-9]|[1][0-2])[./-]([0-9]{4}|[0-9]{2})$"
```

Comment:
Yes, I know it was allowing that, and also any number with 4 digits... Should be improved, specially when we change this to a date picker. I think better to leave this for now since it's just a place holder for another plugin needed. I just wanted to display the use of a validation to check if this is a date.